src/remediations/fifi.js

```diff
@@ -123,32 +163,44 @@ function getName (executor) {
 }
 
 function getStatus (executor) {
-    if (!executor.id) {
+    if (!executor.id && executor.rhcStatus === 'disconnected') {
```

Comment:
again, would be way cleaner to have
```
if (executor.type == "satellite") {
    // do satellite checks
} else if (executor.type == "RHC") {
    // do RHC check if needed
} else {
    // default
}
```

than indirectly inferring whether the given executor is rhc or not from its status.